### PR TITLE
fix: replace mdc with ups

### PR DESCRIPTION
**Pull request by secondsun**
## Motivation
This PR is part of adding UPS to the solution explorer for integr8ly and removing MDC.

## What
This PR removes MDC and adds code for UPS.  It replaces it in the mock data as well as adds hooks for the UPS service to be displayed.

## Verification Steps
1. Deploy integr8ly somewhere
2. Modify the mobile-service-broker to use this image : `quay.io/secondsun/managed-service-broker:master` (a PR of msb is forthcoming) 
3. You can run this PR with the command line call `OPTIONAL_PROVISION_SERVICES=unifiedpush OPENSHIFT_HOST=$YOUR_OPENSHIFT_HOST yarn start:dev` (this will need to be added to the i8ly installer to start the solution explorer with this link)
4. Navigate to localhost:3006 and you should see the push service link.

## Checklist:

- [ x ] Code has been tested locally by PR requester
- [ ] Changes have been successfully verified by another team member

## Progress

- [x] Finished task

### CODE REVIEW

File: src/product-info.js [hunk 1]
```diff
@@ -47,11 +47,11 @@ export default {
...
-  mdc: {
-    prettyName: 'Mobile Developer Console',
-    gaStatus: 'preview',
-    primaryTask: 'Create mobile apps',
-    description: 'Allows you register your mobile app and associate it with Mobile Developer Services.'
+  unifiedpush: {
+    prettyName: 'Push Notification Service',
+    gaStatus: 'GA',
+    primaryTask: 'Mobile Push Messaging',
+    description: 'Provides APNS and FCM messaging services for mobile applications.'
```
Comment (tiffanynolan) [inline src/product-info.js]: Can we make this description more user friendly? I'm not sure general users would know what APNS or FCM mean. Happy tow work with you on this.

**Comment by secondsun** [comment]
Tell me the words and I'll make it happen :)

"mobile push messaging service" or "Provides messaging services for Android
and iOS"?

Summers Pittman
>>Phone:404 941 4698
>>Java is my crack.

**Comment by secondsun** [comment]
@tiffanynolan The RHPDS instance has been retired, but you seemed to be ok with the change minus some word smithing.  If you can let me know what would be a good line for that I'll update the PR and merge so we can get on with starting a release to update the operator and install script.

**Comment by tiffanynolan** [comment]
@secondsun Thanks for the suggested description improvements. I like the second one better. It's simple and to the point. "Provides messaging services for Android and iOS"

**Pull request merged**